client/js/index.js

```diff
@@ -178,9 +179,23 @@ function stopHandler() {
   console.log('stop');
 }
 
+function maxSpeedHandler(inputData) {
+  setSensorValue('powerModelPredMaxSpeed', inputData.predicted_max_speed, 2);
+}
+
+function recSPHandler(inputData) {
+  setSensorValue('powerModelRecSpeed', inputData.rec_speed, 2);
+  setSensorValue('powerModelRecPower', inputData.rec_power, 2);
```

Comment:
Could probably be rounded to no decimal places. Power meters aren't nearly this accurate, nor is the rider's ability to maintain a steady power, nor is the accuracy of the power model

Reply:
That is true. I was just using this as a debugging tool so I can start to write tests for BOOST